tests/test_pseudomotor.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 29 10:18:23 2024
+
+@author: renebes
+"""
+
+from borealis.controller.controller_base import DummyCtrl
+from borealis.motor import Motor
+from borealis.pseudo_motor import PseudoMotor
+from borealis.detector.detector_base import DummyDet
+
+
+def test_pseudomotor_const():
+    ctrl = DummyCtrl()
+    mot1 = Motor('DummyMotor1', '1', 0, ctrl)
+    mot2 = Motor('DummyMotor2', '2', 0, ctrl)
+    geo1 = lambda x: x
+    geo2 = lambda x: -x
+    PseudoMotor('DummyPseudoMotor', [mot1, mot2], [geo1, geo2])
+
+def test_pseudomotor_where():
```

Comment:
It would be nice to add a word on the purpose of those tests.  E.g.:
```suggestion
def test_pseudomotor_where():
    """Check the outputs of the tested functions."""
```

Reply:
Comment and docstring added to explain the purpose of the added tests.